src/qt/walletmodel.h

```diff
@@ -217,6 +217,8 @@ class WalletModel : public QObject
     int getDefaultConfirmTarget() const;
 
     bool getDefaultWalletRbf() const;
+    
+    bool getDefaultBech32() const;
```

Comment:
Expose address type instead `getDefaultAddressType()`?

Reply:
Done

Reply:
Although it introduces a compiler warning:

```
qt/receivecoinsdialog.cpp:45:62: warning: comparison of constant 'OUTPUT_TYPE_BECH32' (3)
with expression of type 'bool' is always false [-Wtautological-constant-out-of-range-compare]
```

Reply:
Nvm, I broke it.